jupyterlite_config.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Thanks! It helped finding the configuration file, but I'm running into other issues now.

To build it locally, I had to run the `jupyter lite build` command manually (with the same options than those set by jupyterlite-sphinx). Running it through sphinx-build was hanging the process (I suspect because of conda/mamba prompt related to the tmp directory into which xeus-python-kernel installs the environment).

Running the command manually and locally still yields this error:

```
Execution aborted.
Two different tasks can't have a common target.'/Users/bbovy/Git/github/benbovy/ipytone/docs/build/html/lite/extensions/ipytone/package.json' is a target for pre_build:federated_extensions:copy:ext:ipytone and pre_build:jupyterlite-xeus-python:copy:ext:ipytone.
```

on RTD build logs seem ok, but I haven't succeed yet to run `import ipytone` with jupyterlite deployed.

Reply:
> on RTD build logs seem ok, but I haven't succeed yet to run import ipytone with jupyterlite deployed.

Using the retrolite directive with a path to a notebook, there's a 404 error 

<img width="766" alt="Screenshot 2022-05-31 at 19 59 00" src="https://user-images.githubusercontent.com/4160723/171254249-78e199c4-e144-42ad-8303-02a4df99ae76.png">

I see the same error on https://ipycanvas.readthedocs.io, actually. Maybe it is the same issue?

Reply:
It's probably related yes. I'll have a look.

Reply:
I finally got it working locally (having disabled `jupyterlite-xeus-python` for now) by using package urls from anaconda.org for the federated extensions (i.e., the changes in https://github.com/jupyterlite/jupyterlite/pull/644). I'm eagerly waiting for the next Jupyterlite release now :)